v1.2.2.md

```diff
@@ -0,0 +1,133 @@
+- [backport v1.2] [BUG] Restore from snapshot not work if target VM is restore-replaced from backup [#4894](https://github.com/harvester/harvester/issues/4894)
+- [backport v1.2] [BUG] switch harvester UI to use rancher user id as backup  when local username is empty during PCIDeviceClaim creation [#4817](https://github.com/harvester/harvester/issues/4817)
+- [backport v1.2] [BUG] Unable to use image that has a name more than ~40 characters long created through terraform [#4799](https://github.com/harvester/harvester/issues/4799)
+- [backport v1.2] [BUG] Can't use data disk measured in terabytes v1.2.1 [#4793](https://github.com/harvester/harvester/issues/4793)
+- [backport v1.2] [BUG] Harvester v1.2.1 stuck on boot with BOSS-S1 hardware RAID [#4791](https://github.com/harvester/harvester/issues/4791)
+- [backport v1.2] [BUG] Resizing Extra Volumes Fail if VM was shutdown via SSH `sudo shutdown -h now` (w/ 422: `admission webhook denied the request: please stop the VM before resizing volumes`) [#4779](https://github.com/harvester/harvester/issues/4779)
+- [backport v1.2] [BUG] The installer occasionally doesn't run [#4771](https://github.com/harvester/harvester/issues/4771)
+- [backport v1.2] [BUG] Avoid to delete LH settings [#4765](https://github.com/harvester/harvester/issues/4765)
+- [BUG] After scale down RKE2/K3s cluster, the deleting node may stuck on "Reconciling" for 10~20m before dropped and created pod will missing then. [#4636](https://github.com/harvester/harvester/issues/4636)
+- [BUG]  Support Bundle missing metadata information in metadata.yaml [#4630](https://github.com/harvester/harvester/issues/4630)
+- [BUG] Harvester v120 cann't start new VM from saved VM template [#4589](https://github.com/harvester/harvester/issues/4589)
+- [BUG] Unable to restore backup into new VM when the name starts with upper case [#4544](https://github.com/harvester/harvester/issues/4544)
+- [BUG] invalid MCM doc link on the Harvester support page [#4539](https://github.com/harvester/harvester/issues/4539)
+- [BUG] Unable to install to MBR disks [#4515](https://github.com/harvester/harvester/issues/4515)
+- [BUG] MTU is overwrite by storage network [#4297](https://github.com/harvester/harvester/issues/4297)
+- [BUG] SR-IOV Virtual Functions can be 0 or -N integers [#3977](https://github.com/harvester/harvester/issues/3977)
+- [EPIC] Add further pre and post checks in harvester upgrade [#3067](https://github.com/harvester/harvester/issues/3067)
+
+
+## Others
+
+
+## Known Issues
+
+## Component Versions
+
+| Component | Version |
+| --- | --- |
+| Longhorn | [v1.5.5](https://github.com/longhorn/longhorn/releases/tag/v1.5.5) |
+| KubeVirt | [v1.1.1](https://github.com/kubevirt/kubevirt/releases/tag/v1.1.1) |
+| Embedded Rancher | [v2.8.2](https://github.com/rancher/rancher/releases/tag/v2.8.2) |
+| RKE2 | [v1.26.15+rke2r1](https://github.com/rancher/rke2/releases/tag/v1.26.15%2Brke2r1) |
```

Comment:
As discussed, let's add SLE Micro dependency here, as Harvester is kind of a variant version of it.

Reply:
Fixed